.github/scripts/fetch_prs.mjs

```diff
@@ -12,72 +12,97 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 import { Octokit } from '@octokit/core';
 import axios from 'axios';
 import moment from 'moment';
 
 const owner = 'deckhouse';
 const repo = 'virtualization';
-const project = 'DVP'
-
+const project = 'DVP';
+const defaulLogin = 'NEED ATTENTION, reviewers are required';
 const octokit = new Octokit({ auth: process.env.RELEASE_PLEASE_TOKEN });
-
+const recentDays = 2;
 
 async function fetchPullRequests() {
-  const pulls = await octokit.request('GET /repos/{owner}/{repo}/pulls', {
-    owner,
-    repo,
-    state: 'all'
-  });
-  return pulls.data.filter(pr => !pr.draft);
+  try {
+    const { data } = await octokit.request('GET /repos/{owner}/{repo}/pulls', {
+      owner,
+      repo,
+      per_page: 500,
+      state: 'open',
+    });
+    return data.filter(pr => !pr.draft);
+  } catch (error) {
+    console.error('Error fetching pull requests:', error);
+    throw error;
+  }
 }
 
-const recentDays = 2;
+async function fetchReviewerDetails(login) {
+  try {
+    const { data } = await octokit.request('GET /users/{username}', {
+      username: login,
+    });
+    return data;
+  } catch (error) {
+    console.error(`Error fetching details for reviewer ${login}:`, error);
+    return null;
+  }
+}
 
-function generateSummary(prs) {
+async function formatPR(pr) {
+  const reviewers = await Promise.all(
+    pr.requested_reviewers.map(async reviewer => {
+      const details = await fetchReviewerDetails(reviewer.login);
+      return details && details.login ? details.login : defaulLogin;
```

Comment:
```suggestion
      return details && details.login ? `${details.login} (${details.name})` : defaulLogin;
```

`name` should be visible and it may help us to determine whom the login belongs.